tests/crash/run.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+trace() {
+  (
+    set +e
+    python ../../strace.py -e execve $1; ec=$?
+    if [ $ec -gt 0 ]; then
+      exit $(($ec - 128 - $2))
+    fi
```

Comment:
Why not writing the tests in pure Python in test_strace.py?

Reply:
I thought that having the possibility to run these tests outside the python testing framework was useful (either for dev of for comprehension). If not, tell me, I will rewrite them in pure Python.

Reply:
I don't feel the need, it's up to you. I'm trying to not be a control freak :-)